ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/JsonRpcErrorHandler.java

```diff
@@ -61,6 +62,12 @@ public void handle(final RoutingContext context) {
             requestId,
             statusCode,
             JsonRpcError.CONNECTION_TO_DOWNSTREAM_NODE_TIMED_OUT);
+      } else if (failure instanceof IllegalStateException
+          && statusCode == HttpResponseStatus.FORBIDDEN.code()) {
+        LOG.error("CORS Rejected", failure);
```

Comment:
Should this be an ERROR level? From what I understand the application is doing the right thing.

Reply:
good catch, this should be a debug or TRACE. I'll fix it.